### PR TITLE
Fix building

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by jreidinger** [comment]
@besser82 JFYI this is revert of line I mentioned in https://github.com/yast/yast-devtools/pull/88#discussion_r30305328

**Comment by lslezak** [comment]
LGTM
@jreidinger thanks for looking into it!

**Comment by jreidinger** [comment]
one more fix goes there

**Comment by mvidner** [comment]
There is still
> error: File not found by glob: /home/abuild/rpmbuild/BUILDROOT/yast2-3.1.122-1.x86_64/usr/lib64/YaST2/*

**Comment by mvidner** [comment]
LGTM, thanks!

File: package/yast2-devtools.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu May 14 08:35:30 UTC 2015 - jreidinger@suse.com
+
+- revert incompatible change for vardir as it break yast2.rpm
+  build
```
Comment (kobliha) [inline package/yast2-devtools.changes]: Still missing some ID :(

Comment (jreidinger) [inline package/yast2-devtools.changes]: same as @besser82 changes, so if it use old version nothing needed to tested and only if change with ID appear, it deserve submission to SP1

**Pull request merged**